mobilecoind/clients/python/blockchain_explorer/blockchain_explorer.py

```diff
@@ -1,9 +1,14 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
 # Copyright (c) 2018-2020 MobileCoin Inc.
 
-import argparse, datetime, sys
+import argparse, datetime
 from flask import Flask, render_template
 
-from mobilecoin import Client
+import os,sys
+sys.path.insert(1, os.path.realpath(os.path.join(os.path.pardir, "lib")))
```

Comment:
This allows us to test the package code in `lib` without pushing a release to pypi.